BaseFilter/sections/allowlist_stealth.txt

```diff
@@ -107,6 +107,8 @@
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/141644 [Stealth Mode - Referrer]
+@@||googleapis.com/customsearch/v1/siterestrict?$domain=rakuten.com,stealth
```

Comment:
There is a mistake in the domain. It should be
```adblock
@@||googleapis.com/customsearch/v*/siterestrict?$stealth,domain=rakuten.com.tw
```

Reply:
Please just a sec.